cips/cip-6.md

```diff
@@ -14,13 +14,17 @@ created: 2023-11-30
 
 Implement a global, consensus-enforced minimum gas price on all transactions. Ensure that all transactions can be decoded and have a valid signer with sufficient balance to cover the cost of the gas allocated in the transaction. The minimum gas price can be modified via on-chain governance.
 
+| Parameter     | Default | Summary                                                                                                                | Changeable via Governance |
+|---------------|---------|------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| gas.MinimumGasPrice | 0.002utia  | Globally set minimum price per unit of gas                                                            | True                     |
```

Comment:
```suggestion
| minfee.MinimumGasPrice | 0.002utia  | Globally set minimum price per unit of gas                                                            | True                     |
```

I think we were going to call the module `minfee`. Also it still seems like there's debate on what the initial/default should be but we can leave it at 0.002 for now